CMakeLists.txt

```diff
@@ -179,8 +179,18 @@ IF(CPUINFO_SUPPORTED_PLATFORM)
     ELSEIF(CMAKE_SYSTEM_NAME MATCHES "^(Windows|WindowsStore|CYGWIN|MSYS)$")
       LIST(APPEND CPUINFO_SRCS src/x86/windows/init.c)
     ENDIF()
+  ELSEIF(CMAKE_SYSTEM_PROCESSOR MATCHES "^(ppc64le)$" )
+	  LIST(APPEND CPUINFO_SRCS
+		  src/powerpc/cache.c
+		  src/powerpc/uarch.c
+		  src/powerpc/linux/chipset.c
+		  src/powerpc/linux/clusters.c
+		  src/powerpc/linux/cpuinfo.c
+		  src/powerpc/linux/hwcap.c
+		  src/powerpc/linux/init.c
+		  src/powerpc/linux/ppc64-isa.c)
   ELSEIF(CMAKE_SYSTEM_NAME MATCHES "^Windows" AND CPUINFO_TARGET_PROCESSOR MATCHES "^(ARM64|arm64)$")
-    LIST(APPEND CPUINFO_SRCS
+	  LIST(APPEND CPUINFO_SRCS
```

Comment:
Why this change is needed?

Reply:
I have removed extra space, now this file follow the flowing indentation